corehq/apps/domain/auth.py

```diff
@@ -64,6 +65,8 @@ def determine_authtype_from_header(request, default=DIGEST):
     elif auth_header.startswith('digest '):
         # Note: this will not identify initial, uncredentialed digest requests
         return DIGEST
+    elif auth_header.startswith('bearer '):
+        return OAUTH2
```

Comment:
One notable is that the API auth flow is the only caller of `determine_authtype_from_header` that actually can handle this format. Which means that if a phone tries to submit via oauth they will get a 500 error. We could also handle that more explicitly.